### DISCUSSION TITLE
Legend of Kyrandia: 3 VMs are wrong

### DISCUSSION THREAD

**Discussion by Vort** [bochs-emu/Bochs#108] [Show and tell]
Hello.

I have discovered that Bochs produces artifacts after starting Legend of Kyrandia game.
Then tested 2 more VMs (VirtualBox and VMware) and found that they have problems as well.
It is probably pain to fix this glitch, so I'm not making issue about it.
But also I do not want to throw away this information.

This is what Bochs and VirtualBox displays:

https://github.com/bochs-emu/Bochs/assets/1242858/c0402b9c-7691-4046-9545-c40e32783515

And this is how it should look like:
https://www.youtube.com/watch?v=PQuBkU3SzxY

**Comment by stlintel** [comment]
nice to see they are almost the same speed :)

**Comment by Vort** [comment]
e3eb603ad4ea64fb2d19b061b358ccd5f2ac6525 solves problems with animation.
